docker-compose.yml

```diff
@@ -26,8 +29,6 @@ services:
     image: mysql:5.6
     ports:
        - "3307:3306"
-    networks:
```

Comment:
Why did we remove the `networks `config?

Reply:
removed this in the multi container setup but for this config I reverted it back since were not using the default network